learn/guides/distinctive-language-features/data.md

```diff
@@ -26,11 +26,12 @@ Broadly, all Ballerina values belong to exactly one kind of basic type. These ar
 
 Simple types are nil, boolean, integers, floating points, and decimal. These are always plain data.
 
-Sequence types are string and xml. These are also always plain data.
+Sequence types are string and XML. These are also always plain data.
 
 Structural types are array, map, record, tuple, and table. These are plain data only if their members are plain data.
 
-And finally, the behavioral types include the function, object, error, stream, typedesc, and handle types. These are not plain data.
+And finally, the behavioral types include the `function`, `object`, `error`, `stream`, `typedesc`, and `handle` types. 
```

Comment:
Should we bold these too be consistent @MaryamZi

Reply:
Just the backticks is enough, right? Bold is usually where it is something significant to the example.